plugin/testing_write_migrate.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func tableUUIDSuffix() string {
-	return strings.ReplaceAll(uuid.NewString(), "-", "_")
+	return strings.ReplaceAll(uuid.NewString(), "-", "_")[:8] // use only first 8 chars
```

Comment:
You don't need the `ReplaceAll` if you're only using the first part of the UUID (which is 8 chars before the first `-`)